### PR TITLE
improve-chart-of-consumption

### LINKED ISSUE #5418: Enhancements to "Rapport graphique de la consommation de stock"
There are three types of consumption in BHIMA : 
- Distribution to patient
- Distribution to service
- Distribution to depot (if the depot that send stocks is a warehouse)

The chart can be more useful if we can distinguish each of these types of consumption, as proposal we can have a multi bar chart where each bar is for a type of consumption

### PR DESCRIPTION
**Pull request by mbayopanda**
Close #5441 
Close #5418 

image

This PR contains only changes about stock consumption graph

### CODE REVIEW

File: client/src/i18n/en/stock.json [hunk 1]
```diff
@@ -241,6 +241,7 @@
     "ENABLE_STRICT_DEPOT_DISTRIBUTION_HELP_TEXT" : "Limits the depots a user can interact with to the depots that the user has access to. Thus, if a user cannot distribute stock from a depot, they also cannot transfer stock to that depot."
   },
   "STOCK_FLUX" : {
+    "ALL_DESTINATION"  : "All Destination",
```
Comment (jniles) [inline client/src/i18n/en/stock.json]: This is better written "All Destinations".

**Comment by mbayopanda** [comment]
@jniles I create a new PR for the chart of consumption, the previous one was corrupted during a rebase

**Review by jniles — COMMENTED** [review]
@mbayopanda, these changes look good.  As demonstrated together, we need to ensure that the caches of the browser are cleared, or that we remove the fiscal year/ period ids on the server.

**Review by jniles — APPROVED** [review]
This looks really nice!  Great work @mbayopanda.  I know the ChartJS API can be hairy - you did a good job making the code readable.

It would be nice to order by value from largest to smallest, but I will create an issue for that.  With graphical reports, we can always improve, improve, improve!

bors r+

**Pull request merged**